app/src/i18n/locales/en.json

```
@@ -91,8 +91,9 @@
     "confirm_app_default": "Are you sure you want to make this app default?",
     "confirm_change_maindomain": "Are you sure you want to change the main domain?",
     "confirm_delete": "Are you sure you want to delete {name}?",
-    "confirm_firewall_allow": "Are you sure you want to open port {port} (protocol: {protocol}, connection: {connection})",
+    "confirm_firewall_allow": "dAre you sure you want to open port {port} (protocol: {protocol}, connection: {connection})",
     "confirm_firewall_disallow": "Are you sure you want to close port {port} (protocol: {protocol}, connection: {connection})",
+    "confirm_group_add_access_permission": "Are you sure you want to grant a {perm} access to your server to {name}'?",
```

```suggestion
    "confirm_group_add_access_permission": "Are you sure you want to grant {perm} access to {name} ? Such access significantly increases the attack surface if {name} happens to be a malicious person. You should only do so if you TRUST this person/group.",
```

Naively I'd put this kind of message, idk